admin/public/styles/keystone/list.less

```diff
@@ -282,7 +282,7 @@
 			width: 16px;
 			height: 18px;
 			padding: 0px;
-			background: url('/keystone/images/icons/list-controls.png') top left no-repeat;
+			background: url('../images/icons/list-controls.png') top left no-repeat;
```

Comment:
this won't work, because pages in the Admin UI are not at a consistent depth. This file needs to be dynamically built with the path to the Admin UI in it when Keystone starts...

unless, @jossmac, could we use less's data-uri feature here to get around this issue?